src/syft/lib/action/plan_wrapper.py

```diff
@@ -0,0 +1,59 @@
+# stdlib
+from typing import List
+from typing import Optional
+
+# third party
+from google.protobuf.reflection import GeneratedProtocolMessageType
+
+# syft absolute
+from syft.core.node.common.plan.plan import Plan
+
+# from syft.core.node.common.action.common import Action
+# from syft.proto.core.node.common.action.action_pb2 import Action as Action_PB
+from syft.proto.core.node.common.plan.plan_pb2 import Plan as Plan_PB
+
+# syft relative
+from ...core.common.uid import UID
+from ...core.store.storeable_object import StorableObject
+from ...util import aggressive_set_attr
+
+
+class PlanWrapper(StorableObject):
```

Comment:
This is the most mysterious part of this PR for me - is this the pointer or a wrapper or... what's the reason for this guy? 

Put another way - is there a reason why Plan doesn't have all these attributes? Maybe I'm forgetting something.

Either way - loads of good documentation explaining what/why this is would be fantastic.

Reply:
Wrappers for the old StorableObject API have all been removed in `dev` as of the weekend so the serde interface is now pretty much just two functions serialize and deserialize plus a proto.